source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/registration/CardRendererRegistration.java

```diff
@@ -76,6 +77,7 @@ private CardRendererRegistration()
         registerRenderer(AdaptiveCardObjectModel.CardElementTypeToString(CardElementType.ChoiceSetInput), ChoiceSetInputRenderer.getInstance());
 
         // Register Action Renderer
+        registerRenderer(AdaptiveCardObjectModel.CardElementTypeToString(CardElementType.ActionSet), ActionSetRenderer.getInstance());
```

Comment:
>registerRenderer(AdaptiveCardObjectModel.CardElementTypeToString(CardElementType.ActionSet), ActionSetRenderer.getInstance()); [](start = 8, length = 126)

Should this be under the "Register Action Renderer" comment section? An ActionSet isn't really itself an action, i would expect it to be sorted with the other body elements (textblock, etc)